File: admin/src/main/java/com/weicoder/admin/po/Logs.scala
package com.weicoder.admin.po

import javax.persistence.Entity
import org.hibernate.annotations.DynamicInsert
import org.hibernate.annotations.DynamicUpdate
import org.springframework.context.annotation.Scope
import org.springframework.stereotype.Component
import org.springframework.beans.factory.config.ConfigurableBeanFactory
import com.weicoder.site.entity.base.BaseEntityIdTime
import scala.beans.BeanProperty

/**
 * 操作日志实体
 * @author WD
 * @since JDK7
 * @version 1._ 2_11-_4-_3
 */
@Component
@Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
@Entity
@DynamicInsert
@DynamicUpdate
class Logs extends BaseEntityIdTime {
  // 内容
  @BeanProperty
  var content: String = _
  // 用户ID
  @BeanProperty
  var userId: Integer = _
  // 状态
  @BeanProperty
  var state: Integer = _
  // 名称
  @BeanProperty
  var name: String = _
  //操作IP
  @BeanProperty
  var ip: String = _
}

File: admin/src/main/java/com/weicoder/admin/po/Logs.java
package com.weicoder.admin.po;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import lombok.Data;

/**
 * 操作日志实体
 * 
 * @author WD
 */
@Entity
@Data
public class Logs {
	// ID
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int id;
	// 名称
	private String name;
	// 内容
	private String content;
	// 操作IP
	private String ip;
	// 时间
	private Integer time;
}